src/Controller/ProductController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Product;
+use App\Repository\ProductRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use JMS\Serializer\SerializerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use FOS\RestBundle\Controller\Annotations\Get;
+use FOS\RestBundle\Controller\Annotations\Post;
+
+class ProductController extends AbstractController
+{
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    public function __construct(SerializerInterface $serializer)
+    {
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * @Get(
+     *     path = "/products/{id}",
+     *     name = "product_show",
+     *     requirements = {"id": "\d+"}
+     * )
+     */
+    public function getProductAction(Product $product)
+    {
+        $data = $this->serializer->serialize($product, "json");
```

Comment:
Those 4 lines could be simplified: `return $this->json($data);` thanks to Symfony ;)

Reply:
Done, but now I have some \n in my json response on Postman